lib/src/gainmapmath.cpp

```diff
@@ -267,6 +286,24 @@ Color hlgInvOetfLUT(Color e_gamma) {
   return {{{hlgInvOetfLUT(e_gamma.r), hlgInvOetfLUT(e_gamma.g), hlgInvOetfLUT(e_gamma.b)}}};
 }
 
+Color hlgOotf(Color e, LuminanceFn luminance) {
+  float y = luminance(e);
+  return e * std::pow(y, 1.2f - 1.0f);
```

Comment:
(This and following methods) I believe we should make a parameter `ootfLamda` instead of a magic number.

Reply:
Gamma = 1.2+0.42×log(LW/1000). As mastering display peak brightness by default is 1000 and this is not modifiable we are currently using 1.2 directly. 
.https://cs.android.com/android/platform/superproject/main/+/main:frameworks/av/media/libstagefright/renderfright/gl/ProgramCache.cpp;l=303

Reply:
Yes but still recommend replacing this with a parameter, like what we did for `globalToneMap()` before instead of magic number flying around

Reply:
Yes, it is better that way, instead of using a magic constant. Done.